modules/prebid/ortb2blocking/hook_bidderrequest.go

```diff
@@ -16,11 +16,11 @@ func handleBidderRequestHook(
 	cfg config,
 	payload hookstage.BidderRequestPayload,
 ) (result hookstage.HookResult[hookstage.BidderRequestPayload], err error) {
-	if payload.BidRequest == nil {
+	if payload.RequestWrapper == nil || payload.RequestWrapper.BidRequest == nil {
 		return result, hookexecution.NewFailure("empty BidRequest provided")
```

Comment:
Please update the error message to a more generic wording, to no longer reference a specific variable name.

Reply:
It's harder than it seems. Is "empty input provided" ok?